src/Microsoft.Identity.Web.TokenAcquisition/MsAuth10AtPop.cs

```diff
@@ -57,14 +62,26 @@ internal static class MsAuth10AtPop
                 { "pop_jwk", jwkClaim }
             };
 
+            var signingCredentials = new X509SigningCredentials(certificate);
             var securityTokenDescriptor = new SecurityTokenDescriptor
-            {
+            { 
                 Claims = claims,
-                SigningCredentials = new X509SigningCredentials(certificate)
+                SigningCredentials = signingCredentials
             };
 
-            var handler = new JsonWebTokenHandler();
-            return handler.CreateToken(securityTokenDescriptor);
+            JwtSecurityTokenHandler tokenHandler = new JwtSecurityTokenHandler();
+            JwtSecurityToken token = (JwtSecurityToken)tokenHandler.CreateToken(securityTokenDescriptor);
+
+            if (sendX5C)
+            {
+                string exportedCertificate = Convert.ToBase64String(certificate.Export(X509ContentType.Cert));
```

Comment:
Does this means that the certificate needs to be exportable?
It's only the public key?

Reply:
The certificate is still protected by the password. See the unit test where I load a cert by the x5c value and I have to provide a password. 

Anyway, I switched to use `GetRawCertData` as this is what MSAL uses for x5c.